ros2/diagnostics/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>diagnostics</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="jlmv.96@gmail.com">jose</maintainer>
```

Comment:
This should ideally be your full name. Also, a valid package.xml requires an `author` tag.